distribution/lib/Standard/Base/0.0.0-dev/src/Data/Index_Sub_Range.enso

```diff
@@ -211,8 +211,8 @@ take_helper length at single_slice slice_ranges range = case range of
         trimmed = resolve_ranges indices length
         slice_ranges trimmed
     Index_Sub_Range.Sample count seed ->
-        rng = Random.new seed
-        indices_to_take = Random.random_indices length count rng
+        Random.set_seed seed
+        indices_to_take = Random.indices length count
```

Comment:
Won't it be safer to do:
```suggestion
        rng = Random.new_generator seed=seed
        indices_to_take = rng.indices length count
```

I see nothing stopping this from working and it will then not suddenly mean the global seed is changed unexpectedly when `Sample` is computed.

Reply:
Done, here and elsewhere.